.github/workflows/compliance-ncs.yml

```diff
@@ -0,0 +1,62 @@
+name: Compliance NCS
+
+on: pull_request
+
+jobs:
+  compliance_job:
+    runs-on: ubuntu-latest
+    name: Run compliance checks on patch series (PR)
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v2
+        with:
+          path: ncs/sdk-zephyr
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0
+
+      - name: cache-pip
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-doc-pip
+
+      - name: Run Compliance Tests
+        env:
+          BASE_REF: ${{ github.base_ref }}
+        working-directory: ncs
+        run: |
+          pip3 install -U pip
+          pip3 install -U setuptools
+          export PATH="$HOME/.local/bin:$PATH"
+          pip3 install -U west
+          west init -m https://github.com/nrfconnect/sdk-nrf
+          west update zephyr nrfxlib
+          pip3 install -U gitlint
+          pip3 install -r zephyr/scripts/requirements-compliance.txt
+          cd sdk-zephyr
```

Comment:
This is checked out in `ncs/zephyr`, not `ncs/sdk-zephyr`

Reply:
No, it is not, check the `checkout` step.